tools/bpf2c/bpf2c.exe.manifest

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+  Copyright (c) Microsoft Corporation
+  SPDX-License-Identifier: MIT
+
+  Overrides the default heap implementation with segment heap.
+-->
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
+  <assemblyIdentity name="bpf2c" type="win32" version="1.0.0.0"></assemblyIdentity>
```

Comment:
Can you please set the assemblyIdentity's name and version correctly? The version should match the semantic versioning of the project (i.e. 0.9.0 currently).

Reply:
Fixed. Thanks @Alan-Jowett